README.md

```diff
@@ -1,38 +1,68 @@
-[![Build Status][travis-image]][travis-url]
-[![Go Report Card][goreportcard-image]][goreportcard-url]
-[![codecov][codecov-image]][codecov-url]
- 
-## Snaps
-
-Package any app for every Linux desktop, server, cloud or device.
-
-Snaps are faster to install, easier to create, safer to run, and they update
-automatically and transactionally so your app is always fresh and never
-broken. You can bring your own build infrastructure or use ours.
-
-Head over to [snapcraft.io](https://snapcraft.io) to get started.
+[![Snapcraft](https://avatars2.githubusercontent.com/u/19532717?s=200)](https://snapcraft.io)
+
+# Welcome to snapd
+
+This is the code repository for **snapd**, the background service that manages
+and maintains installed snaps. 
+
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its various service and management functions, snapd:
+- provides the _snap_ command that's used to install and remove snaps and
+  interact with the wider snap ecosystem
+- implements the confinement policies that isolate snaps from the base system
+  and from each other
+- marshals the interfaces that allow snaps to access specific system resources
+  outside of their confinement
+
+For general details, including
+[installation](https://snapcraft.io/docs/installing-snapd) and [Getting
+started](https://snapcraft.io/docs/getting-started) guides, head over to our
+[Snap documentation](https://snapcraft.io/docs). If you're looking for
+something to install, such as [Spotify](https://snapcraft.io/spotify) or
+[Visual Studio Code](https://snapcraft.io/code), take a look at the [Snap
+Store](https://snapcraft.io/store). And if you want to build your own snaps,
+start with our [Creating a snap](https://snapcraft.io/docs/creating-a-snap)
+documentation.
 
 [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/snapd)
```

Comment:
While at it, I'm not sure we should have a link to the snapd page in snap store as snapd actually comes as a Linux distro package.

Reply:
Yeah, I agree with this.